config/config.sample.php

```diff
@@ -1760,4 +1760,72 @@
  */
 'grace_period.demo_key.link' => 'https://owncloud.com/try-enterprise/',
 
+/**
+ * The order of the login policies that will be checked, if any.
+ * Policies must be registered in order to use / activate them. This is usually
```

Comment:
```suggestion
/**
 * Order of login policies
 * The order of the login policies that will be checked, if any.
 * Policies must be registered in order to use / activate them. This is usually
```
The first line is the headline printed in the TOC on top. It always should be short and without an ending period.